index.html

```diff
@@ -491,7 +491,7 @@ <h1>Exploration Mechanisms</h1>
             <figcaption>The high-level class diagram of the exploration mechanisms</figcaption>
         </figure>
 
-        This specification defines two new Thing Description classes which may be
+        This specification defines two new Thing Description classes in the tdd ontology that may be
```

Comment:
Is "tdd" defined anywhere?

Reply:
it is not defined as a concept, shall we? I though on putting a link to the ontology code. Do you have a suggestion regarding the definition?

Reply:
I defined it as <a>Thing Description Directory</a> ontology